exercises/rotational-cipher/README.txt

```diff
@@ -0,0 +1,71 @@
+# Rotational Cipher
+
+Create an implementation of the rotational cipher, also sometimes called the Caesar cipher.
+
+The Caesar cipher is a simple shift cipher that relies on
+transposing all the letters in the alphabet using an integer key
+between `0` and `26`. Using a key of `0` or `26` will always yield
+the same output due to modular arithmetic. The letter is right-shifted
+the for as many values as the value of the key.
+
+The general notation for rotational ciphers is `ROT + <key>`.
+The most commonly used rotational cipher is `ROT13`.
+
+A `ROT13` on the Latin alphabet would be as follows:
+
+```plain
+Plain:  abcdefghijklmnopqrstuvwxyz
+Cipher: nopqrstuvwxyzabcdefghijklm
+```
+
+It is stronger than the Atbash cipher because it has 26 possible keys, and 24 usable keys.
```

Comment:
0 through 26 inclusive on both ends has 27 elements, rather than 26... but I know you mean that there are 26 *unique* keys since 0 is congruent to 26. It would be good to get some clarity on this point though, in case someone else other than me objects "there are 27 elements in the range!"

There are 25 usable keys - the alignments of (every letter of the alphabet except "a") with "a", rather than 24.